mllib/src/main/scala/org/apache/spark/mllib/tree/model/treeEnsembleModels.scala

```diff
@@ -166,6 +156,56 @@ class GradientBoostedTreesModel(
 
 object GradientBoostedTreesModel extends Loader[GradientBoostedTreesModel] {
 
+  /**
+   * Method to compute initial error and prediction as a RDD for the first
```

Comment:
Doc: "Compute the initial predictions and errors for a dataset for the first iteration of gradient boosting."  No need to say it's a "method."  This is a leftover problem from before, so don't follow previous examples.  : )